flocker/acceptance/scripts/lsblkhttp.py

```diff
@@ -0,0 +1,36 @@
+"""
+HTTP server that returns the size of the mounted volume.
+"""
+
+from subprocess import check_output
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(s):
+        s.send_response(200)
+        s.end_headers()
+        if "is_http" in s.path:
```

Comment:
The name isn't clear what this is for, from the path. I guess it is to determine if the server is ready yet. It would be clearer if the path reflected that.